### ISSUE TITLE
Masternode instability

### ISSUE THREAD

**Issue by unintuitive** [CooleRRSA/crave#5]
After upgrading to v2.0.4.2 I noticed that my masternodes began to stop acting as masternodes after 15-30 minutes of uptime.

The daemon never crashes or becomes unresponsive, but the nodes suddenly drop from the Crave network list. The command

`./craved masternode list full | grep "12.34.56.78\|90.12.34.56"`

no longer returns any results, and the local control wallet also says they are not found in the list. I have to restart the local wallet, unlock it, and start the masternodes again.

After I downgraded to v2.0.4.1, the issue no longer occurs.

Each masternode was built from source on an Ubuntu 16.04 VPS with 1GB memory.

The crave.conf on the VPS is:

```
rpcuser=blahblah
rpcpassword=blahblah
rpcallowip=127.0.0.1
alertnotify=echo %s | mail -s "Crave Masternode Alert" root
listen=1
daemon=1
server=1
staking=0
longtimestamps=1

port=9999
masternode=1
masternodeaddr=myipaddress:9999
masternodeprivkey=blahblah
```

**Comment by Ampakinetic** [comment]
I've had the same problem I think,  I can't get the masternodes to say "masternode started remotely" like they used to before the update with "craved masternode debug".  

I also downgraded to fix the problem in the mean time.

**Comment by CooleRRSA** [comment]
Sorry for that.
I have fixed it in v2.0.4.3
https://github.com/CooleRRSA/crave/commit/8cc20938c51a0bcb30db4c24f29e10212b571b5f

**Comment by Ampakinetic** [comment]
Awesome, thanks for the quick fix, I'll try this now.
